src/NexusMods.App/Services.cs

```diff
@@ -48,6 +49,7 @@ public static IServiceCollection AddListeners(this IServiceCollection services)
         services.AddCLI()
             .AddFileSystem()
             .AddUI()
+            // .AddFomodInstallerUi()
```

Comment:
?

Reply:
Yup, that's not done yet. I didn't want the PR to explode in size, so I'll work on the VMs in the next PR.